### ISSUE TITLE
Navigation buttons

### ISSUE THREAD

**Issue by eBusinessAtlantique** [mageplaza/magento-2-banner-slider#68]
I have a weird behaviour with navigation elements.
Prev and next links always appear under banner 1 even if I set them to be hidden.
Nav dots appear only under banner 1 not under banner 2. But can be hidden if set to non visible.
I would appreciate a hint to solve this.
Thanks !

**Comment by Tuvpt** [comment]
Your problem may be caused by the library and the css file not being loaded yet.
Please run the deploy command and check it again

**Comment by eBusinessAtlantique** [comment]
I am in default mode at the moment and such a thing never happen to any other frontend element.
What can cause that ?
Thanks

**Comment by Tuvpt** [comment]
First, make sure you are using the latest version of our Core module. The problem could be related to the carousel library.
In addition, we may assist you to examine this issue more closely

**Comment by jas8522** [comment]
I upgraded a store from 2.2.9 to 2.4.0, which updated bannerslider and the core module (I double checked) to the latest release. After doing so the next/prev buttons are now under the slider and really tiny (regardless of which slide/banner I'm on). In other words, the CSS that targets these buttons isn't loading. I'm not spotting any errors in static content deployment... I've cleared the cache (even manully by wiping out the folders) many times with no changes.

What should I be checking in the static content deployment process to indicate what's going wrong here such that we're not getting the CSS file loading for this?
